games-arcade/freegish/freegish-1.0.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="A free version of the game Gish"
+DESCRIPTION="FreeGish is an open-source, freely redistributable version of Gish, a \
+physics-based platforming game featuring a ball of tar."
+HOMEPAGE="http://github.com/freegish/freegish"
+COPYRIGHT="2012-2014 Joris van de Donk"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="git://github.com/freegish/freegish.git#f53d915381c2dd9889dbebdff0541b04c6065067"
```

Comment:
It's better to have a link to an archive so it could be considered as safe source  (meaning haikuporter could check its checksum). In this case it will be https://github.com/freegish/freegish/archive/f53d9153.zip